GitHub-Test-3/tyler-gittest3.md

```diff
@@ -0,0 +1,5 @@
+# Git Test 3 Markdown
+
+~~this is git test 7 markdown~~
+
```

Comment:
@tec5271 Nice work with learning your way around markdown. This is an example of a line comment made directly on your commit (you'll need to write one of these for the last part of the Git test). Sorry about the plowing issues in Pittsburgh!